README.md

```diff
@@ -82,7 +82,7 @@ pak::pkg_install("cmu-delphi/epidatr@dev")
 ```
 
 Our CRAN listing is
-[here](https://cran.r-project.org/web/packages/epidatr/index.html).
+[here](https://CRAN.R-project.org/package=epidatr/index.html).
```

Comment:
question: Looks like this link redirects to the old URL. Is there a benefit to doing it this way?

Reply:
cran prefers this format, for whatever reason

Reply:
```

Found the following (possibly) invalid URLs:
  URL: https://cran.r-project.org/web/packages/epidatr/index.html
    From: README.md
    Status: 200
    Message: OK
    CRAN URL not in canonical form
  URL: https://www.cdc.gov/datastatistics/index.html
    From: README.md
    Status: 404
    Message: Not Found
  The canonical URL of the CRAN page for a package is 
    https://CRAN.R-project.org/package=pkgname
```

Reply:
Huh, I guess it's kinda neat that it checks URLs for you.

Reply:
[This page](https://stackoverflow.com/questions/64257455/cran-url-not-in-canonical-form) says the canonical URL is 

`https://cran.r-project.org/package=package-name`

?

Reply:
`urlchecker::url_check()` was satisfied, which gives a more obscure version of the win_builder result, so I think we're good.